crawler.py

```diff
@@ -94,13 +96,18 @@
     sentry_sdk.capture_message("Crawl worker started")
 
 # Connect to job queue
-job_queue = rediswq.RedisWQ(name=REDIS_QUEUE_NAME, host=REDIS_HOST)
+job_queue = rediswq.RedisWQ(
+    name=REDIS_QUEUE_NAME,
+    host=REDIS_HOST,
+    max_retries=MAX_JOB_RETRIES
+)
 manager.logger.info("Worker with sessionID: %s" % job_queue.sessionID())
 manager.logger.info("Initial queue state: empty=%s" % job_queue.empty())
 
 # Crawl sites specified in job queue until empty
 while not job_queue.empty():
-    job = job_queue.lease(lease_secs=120, block=True, timeout=5)
+    job_queue.check_expired_leases()
+    job = job_queue.lease(lease_secs=TIMEOUT + 30, block=True, timeout=5)
```

Comment:
lease_secs need to include DWELL_TIME as well. Otherwise TIMEOUT=10 and DWELL_TIME=60 will wreak havoc :)

Reply:
Good catch :)